packages/directory/app/styles/navi-directory/components/dir-new-button.less

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+.dir-new-button {
+  &__trigger {
+    background-image: linear-gradient(45deg, #3697F2 0%, #3570F4 100%);
```

Comment:
Can we make a generic button mixin that we could add to any button?
And can it be the same as we have in teller? We don't want buttons that look different.

And can we assign variables to these colors?